ibm_catalog.json

```diff
@@ -0,0 +1,211 @@
+{
+  "products": [
+    {
+      "label": "Cloud automation for Monitoring agent",
+      "name": "deploy-arch-ibm-monitoring-agent",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "logging",
+        "logging_monitoring",
+        "terraform",
+        "target_terraform",
+        "solution"
+      ],
+      "keywords": [
+        "monitoring",
+        "agent",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution"
+      ],
+      "provider_name": "IBM",
+      "short_description": "Deploys IBM Monitoring Agent to a cluster",
+      "long_description": "Solutions that support deploying IBM Monitoring Agent for logging.",
```

Comment:
Solutions - > Solution

This agent has nothing to do with logging - its used for metrics - please update